lib/brainstem/api_docs/formatters/open_api_specification/version_2/endpoint/param_definitions_formatter.rb

```diff
@@ -309,3 +281,4 @@ def format_param_branch(branch)
 
 Brainstem::ApiDocs::FORMATTERS[:parameters][:oas_v2] =
   Brainstem::ApiDocs::Formatters::OpenApiSpecification::Version2::Endpoint::ParamDefinitionsFormatter.method(:call)
+
```

Comment:
Do a lot of these files have extra newlines at the end, or am I seeing the diff wrong?

Reply:
This was a mistake. I will fix it.